package.json

```diff
@@ -4,21 +4,22 @@
     "description": "Scripts to use AllureReporter tool from a continuous integration environment",
     "main": "index.js",
     "scripts": {
-        "test": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register",
-        "report": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register --spec src/index.ts --reporter spec --timeout 999999 --colors --full-trace"
+        "report": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register --spec src/index.ts --config .mocharc.json",
+        "report-jenkins": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register --spec src/index.ts --config .mocharc-jenkins.json"
```

Comment:
We do not need this script as reports are always generated.

Reply:
Done